app/helpers/posts_helper.rb

```diff
@@ -0,0 +1,13 @@
+module PostsHelper
+  def is_current_user_owner?
+    if user_signed_in?
```

Comment:
Tu mazliet parsarežginaji condition un current_user ir jau User ieraksts, tāpēc tev nevajag vel vienu pieprasijumu uz datubazi taisit...
Ruby valodai ir tada ipašiba, kā viņa vienmēr atgriež pedejo ierakstu no metodes, tāpēc tev ari return šoreiz nevajadzes.

Daudz skaidrak un skaistak būtu tā:

```ruby
def is_current_user_owner?
  user_signed_in? && current_user.id == @post.user_id
end
```

Šeit ari jābut piesardzigam ar to, kā iekš helperi tu izmanto instances mainigo (kas tas ir, palasi). Kuram it ka vajadzetu būt definetem kontrolerī.
Jā tu izmantosi šo helperi vel kaut kada citā skatā un instances mainigais nebūs definets, tu saņemsi kļudu. 
Tāpēc ari būtu praktiskak padot viņu helperi kā mainigo:
`def is_current_user_owner?(post)`

Pagaidam pedejais nav aktuals. Tagad kad tava sistema augs būs vel grūtāk un grūtāk izsekot lietotaju visas atļautas darbibas. Tāpēc ar vienu no taskiem tev vajadzes uzlīkt Pundit autorizacijas sistemu. Bet tas jau cits tasks